### ISSUE TITLE
Max vertices error

### ISSUE THREAD

**Issue by Hungary-Dude** [libgdx/fbx-conv#46]
After running:

```
fbx-conv -p -v "path to file"
```

FBX-conv gives me a warning that there are too many mesh vertices. So I try setting the maximum allowed indices with this command:

```
fbx-conv -p -v -m 64k "path to file"
```

It prints the warning again and tells me the max is now "64." If FBX-conv does not accept *k numbers then please do not put them in the help printout. So I ran:

```
fbx-conv -p -v -m 64000 "path to file"
```

Now what does it do, it says "Error, maximum vertex count must be between 0 and 32k"
This is annoying, as now I have to go back into my model editor and split up the node.

So in conclusion, please make FBX-conv accept *k numbers, and please make it accept larger vertex counts if possible.

**Comment by JonasVautherin** [comment]
Is it still the case that libgdx uses signed short indices? Is there no way to have more than 32k triangles in a mesh?

**Comment by ferrolho** [comment]
So this explains why I am not able to load complex models... @xoppa, why does libgdx use signed short indices?
Is there any workaround so I can load models like this one: http://tf3dm.com/3d-model/sa342-gazelle-97281.html ?
Thanks!

**Comment by xoppa** [comment]
Because java doesn't support unsigned short. Don't expect random high poly models to work. Use low poly models instead.

**Comment by ferrolho** [comment]
Thanks @xoppa, and Merry Christmas!

**Comment by dpproduction** [comment]
Hi guys, I started use libGDX for our project, but limitation 32k is too small for us.
If I fork engine and change all short values for example to int does it work good?
Or short value is system restriction?  
Can somebody explain me please?
